ui/components/institutional/confirm-remove-jwt-modal/confirm-remove-jwt-modal.js

```diff
@@ -93,39 +94,41 @@ const ConfirmRemoveJWT = ({
       <ModalOverlay />
       <ModalContent>
         <ModalHeader onClose={hideModal}>{t('removeJWT')}</ModalHeader>
-        <Box
-          display={Display.Flex}
-          padding={2}
-          borderRadius={BorderRadius.SM}
-          className="confirm-action-jwt__jwt"
-        >
-          <Text ellipsis>
-            {showMore && token ? token : `...${token.slice(-9)}`}
-          </Text>
-        </Box>
-        {!showMore && (
+        <ModalBody>
+          <Box
+            display={Display.Flex}
+            padding={2}
+            borderRadius={BorderRadius.SM}
+            className="confirm-action-jwt__jwt"
+          >
+            <Text ellipsis>
+              {showMore && token ? token : `...${token.slice(-9)}`}
+            </Text>
+          </Box>
+          {!showMore && (
+            <Text
+              color={TextColor.goerli}
+              marginLeft={2}
+              className="confirm-action-jwt__show-more"
+            >
+              <a rel="noopener noreferrer" onClick={handleShowMore}>
+                {t('showMore')}
+              </a>
+            </Text>
+          )}
           <Text
-            color={TextColor.goerli}
-            marginLeft={2}
-            className="confirm-action-jwt__show-more"
+            as="h6"
+            textAlign={TextAlign.Center}
+            variant={TextVariant.bodySm}
+            marginTop={2}
           >
-            <a rel="noopener noreferrer" onClick={handleShowMore}>
-              {t('showMore')}
-            </a>
+            {t('removeJWTDescription')}
           </Text>
-        )}
-        <Text
-          as="h6"
-          textAlign={TextAlign.Center}
-          variant={TextVariant.bodySm}
-          marginTop={2}
-        >
-          {t('removeJWTDescription')}
-        </Text>
-        <Box className="confirm-action-jwt__accounts-list">
-          <CustodyAccountList accounts={tokenAccounts} rawList />
-        </Box>
-        <Box display={Display.Flex}>
+          <Box className="confirm-action-jwt__accounts-list">
+            <CustodyAccountList accounts={tokenAccounts} rawList />
+          </Box>
+        </ModalBody>
+        <Box display={Display.Flex} padding={4}>
           <Button
             block
             variant={BUTTON_VARIANT.PRIMARY}
```

Comment:
Although it seems like a lot of code changes here I'm just wrapping the elements after the `<ModalHeader>` in a `<ModalBody>` component so the scrolling behaviour doesn't break

### Before
<img width="1511" alt="Screenshot 2023-12-10 at 2 52 53 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/4fb926a7-8ee0-4b7c-951b-fdef47639641">

### After with only imports updates to use deprecated
<img width="1511" alt="Screenshot 2023-12-10 at 2 53 29 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/75808b27-b47d-437f-91d4-e8b258ebeba2">

### After with component updates
<img width="1512" alt="Screenshot 2023-12-10 at 2 57 06 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/94902b6e-a45d-4e95-9c10-c8114a87ee9c">